webapp/components/wpt-results.html

```diff
@@ -503,6 +503,16 @@ <h3>
               };
               this.fetchDiff();
             }
+
+            // Load a manifest.
+            if (this.fetchManifestForTestList) {
+              const shas = (runs || []).reduce((set, run) => {
```

Comment:
Unless this is an optimization, then something like `const shas = runs ? new Set(runs.map(run = > run.revision)) : null` would be easier to follow.

Reply:
kept `(runs || [])` which is more succinct, but took the ctor suggestion.